openml/__init__.py

```diff
@@ -26,4 +26,4 @@
 __version__ = "0.2.1"
 
 __all__ = ['APIConnector', 'OpenMLDataset', 'OpenMLRun', 'OpenMLSplit',
-           'datasets', 'OpenMLTask', 'OpenMLFlow']
+           'datasets', 'OpenMLTask', 'OpenMLFlow', 'runs']
```

Comment:
you need to ``from . import runs`` - actually we need to do the same for flows and tasks... I forgot that it seems.